sig-observability/concepts-and-designs/multi-tenant notification.md

```diff
@@ -62,16 +62,16 @@
 
 ## Requirements ##
 
-- User can define the channels for receiving notifications, including email, wechat, slack, webhook and dingtalk, by creating the corresponding receiver and config.
-- A user only has one receiver or config in the same type.
-- A user can update, delete, get the receiver, and config created by oneself.
-- A user can get the default config.
-- The receiver created by the user can use the config create by the user and the default config.
+- Every user can customize the channels for receiving notifications, including email, wechat, slack, webhook and dingtalk, by customize his own receiver and config. A user only has one receiver or config in the same type.
```

Comment:
Every user can customize the channels for receiving notifications, including email, wechat, slack, webhook and dingtalk, by customizing his own receiver and config. A user can only have one receiver or config of the same type.